probe/kubernetes/daemonset.go

```diff
@@ -48,5 +48,6 @@ func (d *daemonSet) GetNode() report.Node {
 		DesiredReplicas:      fmt.Sprint(d.Status.DesiredNumberScheduled),
 		Replicas:             fmt.Sprint(d.Status.CurrentNumberScheduled),
 		MisscheduledReplicas: fmt.Sprint(d.Status.NumberMisscheduled),
+		NodeType:             "Daemon Set",
```

Comment:
We can add the value in during rendering, though it's not obvious how we can also cleanly add in the metadata_template value so that it actually displays.
It's a fundamental part of scope's design that considerable amounts of display logic comes directly from the scope probe, which means old reports will use old display settings. If that's something we want to move away from, we can, but it will take time, and brings with it complications eg. displaying values that simply don't exist in older reports.